DisplayCAL/madvr.py

```diff
@@ -351,8 +345,8 @@ def __init__(self, stream_or_filename=None, check_lut_size=True):
         self.parametersData = dict()
         for line in (
             data[self.parametersFileOffset: self.parametersFileOffset + parametersSize]
-            .rstrip(b"\0")
-            .splitlines()
+                .rstrip(b"\0")
```

Comment:
formatting changes in this file are the result of running black, no manual formatting changes made

Reply:
I think that is, because at some point we decided to go with the black default maximum line length of 88 characters. However the PEP8 default which seems to be applied by your black configuration seems to be 79 characters I think.